### PR TITLE
Updated quick start README, replaced route guide docs with link to grpc.io

### PR DESCRIPTION
**Pull request by murgatroid99**
This mirrors the changes in #3442.

### CODE REVIEW

File: examples/node/README.md [hunk 1]
```diff
@@ -55,6 +45,5 @@ TUTORIAL
 You can find a more detailed tutorial in [gRPC Basics: Node.js][]
 
 [homebrew]:http://brew.sh
-[linuxbrew]:https://github.com/Homebrew/linuxbrew#installation
-[gRPC install script]:https://raw.githubusercontent.com/grpc/homebrew-grpc/master/scripts/install
-[gRPC Basics: Node.js]:https://github.com/grpc/grpc/blob/master/examples/node/route_guide/README.md
+[Install gRPC Core]:../../INSTALL
```
Comment (tbetbetbe) [inline examples/node/README.md]: This should link to the install for gRPC Node.js

Comment (murgatroid99) [inline examples/node/README.md]: You don't need to install the Node.js library specifically. As long as you have core on your system, you can do `npm install` with a package that depends on grpc (like this examples package), and everything will be installed properly.

Comment (tbetbetbe) [inline examples/node/README.md]: That's not really the point.  
It does not make sense to have [these instructions] (https://github.com/grpc/grpc/tree/master/src/node#installation) for installing gRPC Node, yet link to something else here.  We've had some feedback about this (inconsistent installation directions) and are aiming to fix it for beta.

Comment (murgatroid99) [inline examples/node/README.md]: I changed it.

File: examples/node/README.md [hunk 2]
```diff
@@ -5,21 +5,11 @@ PREREQUISITES
 -------------
 
 - `node`: This requires Node 10.x or greater.
-- [homebrew][] on Mac OS X, [linuxbrew][] on Linux.  These simplify the installation of the gRPC C core.
+- [homebrew][] on Mac OS X.  This simplifies the.
```
Comment (tbetbetbe) [inline examples/node/README.md]: this sentence is incomplete

Comment (murgatroid99) [inline examples/node/README.md]: Oops. Fixed.

**Pull request merged**